ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/CommandlineParser.java

```diff
@@ -39,39 +44,40 @@ public void registerSigner(final SignerSubCommand signerSubCommand) {
     signers.add(signerSubCommand);
   }
 
-  public void parseCommandLine(final String... args) {
+  public boolean parseCommandLine(final String... args) {
 
     final CommandLine commandLine = new CommandLine(baseCommand);
     commandLine.setCaseInsensitiveEnumValuesAllowed(true);
+    // commandLine.setUnmatchedArgumentsAllowed(true);
     commandLine.registerConverter(Level.class, Level::valueOf);
 
     for (final SignerSubCommand subcommand : signers) {
       commandLine.addSubcommand(subcommand.getCommandName(), subcommand);
     }
 
-    // Must manually show the usage/version info, as per the design of picocli
-    // (https://picocli.info/#_printing_help_automatically)
     try {
       commandLine.parseWithHandlers(
           new RunLast().useOut(output), new ExceptionHandler<List<Object>>(), args);
```

Comment:
can use use diamonds for ExceptionHandler type

Reply:
done.